app/src/features/apiClient/commands/collectionRunner/runCollection.command.ts

```diff
@@ -181,8 +186,36 @@ class Runner {
     });
   }
 
+  private populateAutogenerateStore(recordId: string) {
+    const { getData, getParentChain } = this.ctx.stores.records.getState();
+    const apiRecord = getData(recordId);
+
+    if (!apiRecord || !apiRecord.data) {
+      return;
+    }
+
+    const childDetails = {
+      id: apiRecord.id,
+      parentId: apiRecord.collectionId,
+    };
+
+    const resolver = <T extends Record<string, any>>(template: T) => {
+      return renderVariables(template, apiRecord.id, this.ctx).result;
```

Comment:
`renderVariables` would also take `scopes` as an argument. This is needed to render variables from data file in collection runner